conf/timezones.ics

```diff
@@ -3832,6 +3765,26 @@ RRULE:FREQ=YEARLY;WKST=MO;INTERVAL=1;BYMONTH=3;BYDAY=-1SU
 END:DAYLIGHT
 END:VTIMEZONE
 BEGIN:VTIMEZONE
+TZID:Europe/Dublin
```

Comment:
Note I have seen a lot of discussion about the new Dublin timezone.  It probably needs special testing.  Suggest scouting out the tz mailing lists for possible issues.  The main thing is that STANDARD time is now in summer and DAYLIGHT is now in winter.

Reply:
Sure will do that